definition_validator/validator.cc

```diff
@@ -190,6 +192,27 @@ class ValidateWalk {
 private:
     UnorderedMap<core::SymbolRef, vector<core::SymbolRef>> abstractCache;
 
+    struct GenericFrame {
```

Comment:
why do we need this at all, can we rely on owner chain of symbol instead?

Reply:
We can, but I was trying to be sensitive to the number of dereferences that are going to happen to get to the information necessary. I'm happy to remove this and go back to how it was working before.